Multisig/UI/App/UIKit/UI Library/ReusableCells/AddressDetailsCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  AddressCell.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 12.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class AddressDetailsCell: UITableViewCell {
+    @IBOutlet private weak var addressWithTitleView: AddressWithTitleView!
+
+    private var style: Style = .nameAndAddress
+    var onViewDetails: (() -> Void)?
+
+    static let rowHeight: CGFloat = 68
+
+    enum Style {
```

Comment:
I thought about it. I used style here to make it easier to adjust it there will be a third/fourth style.
But with name = nil it could also be achieved. 

Do you think it worth refactoring at this moment?

Reply:
No, I think it's not worth it, the scope is small